site/pages/[locale]/dp-auctions/collections/index.js

```diff
@@ -21,15 +21,15 @@ const ETH_BLOCK_TIME = 13 // Average block time in Ethereum
 // The remaining time is shown in minutes. So every 10 seconds the remaining
 // time is re-calculated.
 const DPAuctionsRow = function ({ auction }) {
-  const { lang, t } = useTranslation('common')
+  const { lang, t } = useTranslations()
 
   const calcEndTime = () =>
     Date.now() +
     (auction.endBlock - auction.currentBlock) * ETH_BLOCK_TIME * 1000
   const endTime = useUpdatingState(calcEndTime, 10000, [auction.currentBlock]) // 10s
 
   return (
-    <Link href={`/dp-auctions/auctions/${auction.id}`} passHref>
+    <Link href={`/dp-auctions/auctions?=id${auction.id}`} passHref>
```

Comment:
Query string is incorrect!

```suggestion
    <Link href={`/dp-auctions/auctions?id=${auction.id}`} passHref>
```